HopsanGUI/LogDataHandler2.cpp

```diff
@@ -899,7 +924,18 @@ void LogDataHandler2::importFromPlainColumnCsv(QString importFilePath, const QCh
     QFileInfo fileInfo(file);
     gpConfig->setStringSetting(CFG_PLOTDATADIR, fileInfo.absolutePath());
 
-    CoreCSVParserAccess csvparser(importFilePath,separator);
+    QTextStream ts(&file);
+    QStringList names;
+    file.open(QFile::ReadOnly);
+    if(linesToSkip > 0) {
```

Comment:
Assumes names on first line, but OK for now I guess, at least better then before

Reply:
Yes, I guess 90+% of all CSV files will have names on first line, and we have to assume something. Only problem would be if same name is used multiple times.

Reply:
yes, but hopsan could "increment" the name in that case, but the question is how much error checking is really necessary, at some point we must assume valid input files. But this could be a nice thing to check, if someone is creating these files by hand it is easy to copy paste something and a check lite this could safe that person some time

Reply:
That's true. I will add a check for unique names.